server/controllers/statistics.ts

```diff
@@ -0,0 +1,23 @@
+import { Request, Response, NextFunction } from 'express';
+import Family from '../database/models/family';
+import Doners from '../database/models/donor';
+import Donations from '../database/models/donation';
+import { sequelize } from '../database/models';
```

Comment:
We already add all these models with sequelize in the index of the models folder, so you can import them together